src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/preview/main.js

```diff
@@ -7,7 +7,7 @@
  * with this source code in the file LICENSE.
  */
 
-define(['app-config'], function(AppConfig) {
+define(['app-config', 'ws-manager'], function(AppConfig, WsManager) {
```

Comment:
Have we already introduced other abbreviations? I thought until now we always used the full names.